### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maxim-belkin**
Homebrew/brew now works on Linux. Updating README.md accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,8 @@ Bundler for non-Ruby dependencies from Homebrew.
 
 ## Requirements
 
-[Homebrew](https://github.com/Homebrew/brew) or [Linuxbrew](https://github.com/Linuxbrew/brew) are used for installing the dependencies.
-Linuxbrew is a fork of Homebrew for Linux, while Homebrew only works on macOS.
-This tool is developed for use with Homebrew on macOS but might work with Linuxbrew (but is unsupported so don't file Linux issues, please).
+[Homebrew](https://github.com/Homebrew/brew) on macOS for installing ependencies.
```
Comment (issyl0) [inline README.md]:
```suggestion
[Homebrew](https://github.com/Homebrew/brew) on MacOS for installing the dependencies.
```

File: README.md [hunk 2]
```diff
@@ -6,9 +6,8 @@ Bundler for non-Ruby dependencies from Homebrew.
...
+This tool might work with Homebrew on Linux but is unsupported so don't file Linux issues.
```
Comment (issyl0) [inline README.md]:
```suggestion
This tool might work with [Homebrew on Linux](https://docs.brew.sh/Homebrew-on-Linux) but is unsupported so don't file Linux issues.
```

File: README.md (hunk 1 above)
Comment (maxim-belkin) [reply README.md]: Thanks, @issyl0. It's macOS, not MacOS. Could you please update the suggestion?

Comment (issyl0) [reply README.md]: Oops, my bad!

**Comment by maxim-belkin** [comment]
Thank you, @issyl0!

**Pull request merged**